boards/tsi/tsi/Kconfig.tsi

```diff
@@ -2,4 +2,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_TSI
-	select SOC_SKYP
+	select SOC_MIMX8ML8_ADSP if BOARD_TSI_MIMX8ML8_ADSP
```

Comment:
I know you mentioned that we had to use MIIMX8ML8 requirement somewhere but since these are new configs we can define these right ? I had defined something similar in the earlier PRs in main branch you can look it up.

Reply:
resolving this comment as I made changes to address this, the new target is "tsi/skyp/txe" now for TXE.